src/main/java/picard/util/IntervalListTools.java

```diff
@@ -495,6 +514,22 @@ private IntervalList openIntervalLists(final List<File> files, BinaryOperator<In
                 .orElse(null);
     }
 
+    private static IntervalList uniqued(IntervalList non_unique, boolean merge_abutting) {
```

Comment:
We tend to use camelCase for variables and methods
```suggestion
    private static IntervalList uniqued(IntervalList nonUnique, boolean mergeAbutting) {
```